deploy/base/rbac.yaml

```diff
@@ -3,6 +3,7 @@ kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: objectstorage-controller-role
+  namespace: default
```

Comment:
It's odd this needs to be added, and not expected. Could you elaborate why these were added? `kustomize` (or `kubectl -k`) should take care of these already.